busybox.yaml

```diff
@@ -97,6 +97,7 @@ subpackages:
 
           mkdir -p "${{targets.subpkgdir}}"/etc/busybox-paths.d
           ./busybox --list-path > "${{targets.subpkgdir}}"/etc/busybox-paths.d/busybox-full
+          ./busybox --install -s
```

Comment:
I don't think we need this - there's an apko feature to do the symlink expansion i think